hepcrawl/testlib/utils.py

```diff
@@ -39,3 +39,8 @@ def deep_sort(element):
         return sorted([deep_sort(item) for item in element])
 
     return element
+
+
+def sort_list_of_records_by_record_title(list_to_sort):
+    """Sort list of records by record title"""
+    return sorted(list_to_sort, key=lambda k: k['titles'][0]['title'])
```

Comment:
why do you need this? how come the order is not deterministic?

Reply:
Apparently it seems that they come in non-determinicstic order. For example for wsp spider the tests are using a file `wsp_smoke_records.json` in which there are two records - the first one has the titles `{'source': 'WSP', 'title': u'Article-title\u2019s'}` and the second one `[{'source': 'WSP', 'title': u'Article-title L\xe9vy char'}]` but in tests the results comes in reverted order [here is the error](https://travis-ci.org/github/inspirehep/hepcrawl/jobs/756428402). The same happens for other spiders and without ordering the list of results they fail.